netlink-request/src/lib.rs

```diff
@@ -26,6 +23,19 @@ mod linux {
         };
     }
 
+    pub fn get_max_netlink_buffer_length() -> usize {
```

Comment:
Not a strong preference but this could just be called `max_netlink_buffer_length()` and the function below could be `max_genl_payload_length()`

Reply:
Updated, also did some variable renames to match.